demo/simple/index.html

```diff
@@ -0,0 +1,23 @@
+<!DOCTYPE html>
+<html lang="en">
+  <meta charset="utf-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1">
+  <title>Virtual Scoller Demos</title>
```

Comment:
Maybe "Simple Demos" to be distinct from https://github.com/WICG/virtual-scroller/pull/173